Makefile

```diff
@@ -118,13 +119,33 @@ $(BUILD_DIR)/$(PROJECT)-%: $(STATIK_FILES) $(GO_FILES) $(BUILD_DIR) deploy/cross
 	(cd `dirname $@`; shasum -a 256 `basename $@`) | tee $@.sha256
 	file $@ || true
```

Comment:
So we could just replace this `cross`, since we won't be using it anymore.  Then there's no kokoro-way and cross-way — there's just the one.

Reply:
I was going to make a follow up PR to remove the code related to our old release pipeline once we can verify that the kokoro way works for us. Would you like me to remove it now?